lib/database/migrations/20231025101611-create-lhc-periods-view.js

```diff
@@ -1,14 +1,28 @@
 'use strict';
 
+const CENTER_OF_MASS_ENERGY_FACTOR_KEY_PREFIX = '2#Z/A';
+
+const basicParticlesProperties = { // 2 * sqrt(ATOMIC_MASS / ATOMIC_NUMBER)
+    [`${CENTER_OF_MASS_ENERGY_FACTOR_KEY_PREFIX}_pp`]: 2 * Math.sqrt(1 / 1),
+    [`${CENTER_OF_MASS_ENERGY_FACTOR_KEY_PREFIX}_PbPb`]: 2 * Math.sqrt(207.2 / 82),
+    [`${CENTER_OF_MASS_ENERGY_FACTOR_KEY_PREFIX}_OO`]: 2 * Math.sqrt(15.9999, 8),
+    [`${CENTER_OF_MASS_ENERGY_FACTOR_KEY_PREFIX}_XeXe`]: 2 * Math.sqrt(131.29, 54),
+    [`${CENTER_OF_MASS_ENERGY_FACTOR_KEY_PREFIX}_cosmic`]: 1,
+};
+
 const CREATE_PERIODS_VIEW = `
 CREATE OR REPLACE VIEW lhc_periods_statistics AS
 SELECT  
     p.id,
-    AVG(r.lhc_beam_energy) AS avgEnergy
+    AVG(r.lhc_beam_energy * center_of_mass_energy_factors.value) AS avg_center_of_mass_energy,
+    GROUP_CONCAT(DISTINCT(r.lhc_beam_energy)) AS distinct_energies
```

Comment:
If not like this I see two other scenerios 
1. Put such a include in `LhcPeriodStatisticsService`
```
queryBuilder.include({
            association: 'lhcPeriod',
            include: [
                {
                    association: 'runs',
                    attributes: [
                        [
                            Sequelize.fn('GROUP_CONCAT', Sequelize.fn('DISTINCT', Sequelize.col('`lhcPeriod->runs`.lhc_beam_energy'))),
                            'distinctEnergies',
                        ],
                    ],
                },
            ],
        });

        queryBuilder.options.group = ['`LhcPeriodStatistics`.`id`'];
```
then adapt `distinctEnergies` (comma separated string) from `lhcPertiod` from `LhcPeriodStatisticsModel` 

2.  Put similar include (without distinct):
```
queryBuilder.include({
            association: 'lhcPeriod',
            include: [
                {
                    association: 'runs',
                    attributes: [
                        'lhcBeamEnergy',
                    ],
                },
            ],
        });
```
and then do unique on lhcBeamEnergies
___
Both are significantly more complex (in terms of computational complexity and code complexity) than current
Or maybe something else?

Reply:
In terms of SQL efficiency I agree than doing the concat here is probably more optimized. But then, you will have to do a lot of split, which is probably worse.
In terms of code complexity, I think that doing group concat then split is worse than puting an include on a column, and you will have directly the right value.
Plus, what happens if at some point you want another property of run?

Reply:
refactored, but in different way that I proposed previously, because those solution were not feasible, due to sequelize problem (bug) with subQuery, limit and ordering..., added required attribute as literal